app/src/main/AndroidManifest.xml

```diff
@@ -9,6 +9,9 @@
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
         android:theme="@style/Theme.CounterApplication">
+        <activity

```

Comment:
remove all other things different to welcome screen

Reply:
done